core/templates/dev/head/pages/collection_editor/editor_tab/CollectionNodeCreatorDirective.js

```diff
@@ -73,6 +90,16 @@ oppia.directive('collectionNodeCreator', [function() {
           );
         };
 
+        var cleanExplorationId = function(newExplorationId) {
```

Comment:
@souravbadami could you please answer @joshuacano's questions? I don't quite understand the aim of this function either.